hca-dcp/submit.wdl

```diff
@@ -0,0 +1,163 @@
+# Get Cromwell metadata for the workflow that produced the given output
+task get_metadata {
+  String analysis_output_path
+
+  command <<<
+
+    # Get workflow_id
+    python <<CODE > workflow_id.txt
```

Comment:
I don't understand how this line is working. specifically the `> workflow_id.txt`

Reply:
it is pipe to stdout :)

Reply:
What it means is "run the python script delimited by 'CODE' and then write the output to a file workflow_id.txt". It seems to be the right way to make this happen, though perhaps not the most intuitive syntax. I originally wanted to add a > after the ending 'CODE' but that didn't work.